src/app/utils/field-validation/answerField.factory.ts

```diff
@@ -0,0 +1,24 @@
+import { BasicField, IColumnSchema } from '../../../types'
+
+type ColumnWithFieldProperties = IColumnSchema & {
```

Comment:
could you add comments to explain what this type is trying to achieve for posterity?

Reply:
added in